pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -231,15 +231,15 @@ func (r *MigPlan) ListMigrations(client k8sclient.Client) ([]*MigMigration, erro
 // Registry
 //
 
-// Registry label for controller-created migration registry resources
+// Registry labels for controller-created migration registry resources
 const (
-	MigrationRegistryLabel = "migration-registry"
+	MigrationRegistryLabel = "migration.openshift.io/migration-registry"
```

Comment:
Are we namespacing labels now too? So far we've only been adding "migration.openshift.io" to annotations. Or is this a new convention we're trying to follow @djwhatle

Reply:
Slowly moving towards namespacing labels, see https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ for some sample labels